Plugins/GeoModel/include/Acts/Plugins/GeoModel/SplitModulesByRadius.hpp

```diff
@@ -0,0 +1,43 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2024 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#pragma once
+
+#include "Acts/Plugins/GeoModel/GeoModelDetectorElement.hpp"
+#include "Acts/Utilities/Logger.hpp"
+
+#include <map>
+#include <string>
+#include <vector>
+
+namespace Acts {
+
+class ModuleByRadiusSplitter {
```

Comment:
Is this class used anywhere other than the python bindings?

Reply:
Nope. Do you think I should implement it directly in the bindings/in python? I though about it, but it seemed to me there would be a lot of code binding necessary for this...

Reply:
Not necessarily, no. I was just wondering about this. Should this be called in a unit test at least somewhere?